src/Sulu/Bundle/ContactBundle/Resources/public/js/services/account-manager.js

```diff
@@ -439,6 +439,32 @@ define([
         },
 
         /**
+         * Sets medias to an account. Currently associated medias get replaced.
+         * @param mediaIds medias to associate with the contact
+         * @param accountId The account to set the medias to
+         */
+        setDocuments: function(accountId, mediaIds) {
+            var promise = $.Deferred();
+
+            Util.ajax({
+                url: '/admin/api/accounts/' + accountId + '/medias',
+                data: {mediaIds: mediaIds},
+                type: 'PUT',
+
+                success: function() {
+                    Mediator.emit('sulu.contacts.account.documents.saved', accountId, mediaIds);
+                    Mediator.emit('sulu.labels.success.show', 'contact.accounts.documents-saved');
```

Comment:
I guess these events are also already emitted in other parts of this manager, right? I am asking because they should actually not be emitted here 😕